.codecov.yml

```diff
@@ -33,6 +33,7 @@ ignore:
   - "LICENSES"
   - "*_test.go"
   - "marker.go" # This file only contains empty function stub
+  - "http.go" # Ignore the http request handler
```

Comment:
Unlike `marker.go` the `http.go` does contain logic and I think it should be tested eventually (by writing an `http_test.go`). Please remove this line.